tools/packaging/packaging.psm1

```diff
@@ -1270,17 +1270,11 @@ function Get-PackageDependencies
                 "libgssapi-krb5-2",
                 "liblttng-ust0",
                 "libstdc++6",
-                "zlib1g"
+                "zlib1g",
+                "libicu72|libicu71|libicu70|libicu69|libicu68|libicu67|libicu66|libicu65|libicu63|libicu60|libicu57|libicu55|libicu52",
+                "libssl1.1|libssl1.0.2|libssl1.0.0"
```

Comment:
with this change it looks like we are eliminating the unsupported error when faced with a platform we don't understand. Does that mean we are relying on an error generated when the dependency is not found?

Reply:
yes, basically, but that was only a build time error.  Since the attempt here is to build a universal, package, keeping the error didn't make since.

If we hit a platform that has a missing dependency, then we should add it to the list if supported or not if the platform is not.